test/runtests.jl

```diff
@@ -197,6 +199,80 @@ end
     @test @inferred(f9(plan_fft(zeros(10), 1), 10)) == 1/10
 end
 
+@testset "output size" begin
+    @testset "complex fft output size" begin
+        for x in (randn(3), randn(3, 4), randn(3, 4, 5))
+            N = ndims(x)
+            y = randn(size(x))
+            for dims in unique((1, 1:N, N))
+                P = plan_fft(x, dims)
+                @test AbstractFFTs.output_size(P) == size(x)
```

Comment:
Maybe also check if the new functions can be inferred?
```suggestion
                @test @inferred(AbstractFFTs.output_size(P)) == size(x)
```
etc.